docs/gen-docs/kyma_alpha_create_module.md

```diff
@@ -20,9 +20,11 @@ Alternatively, a custom (non kubebuilder) module can be created by providing a p
 
 Optionally, you can manually add additional layers with contents in other paths (see [resource flag](#flags) for more information).
 
-Finally, if you provided a registry to which to push the artifact, the created module is validated and pushed. For example, the default CR defined in the \"default.yaml\" file is validated against CustomResourceDefinition.
+Finally, if you provided a registry to which to push the artifact, the created module is validated and pushed. During the validation the default CR defined in the optional \"default.yaml\" file is validated against CustomResourceDefinition.
+Alternatively, you can trigger an on-demand default CR validation with "--validateCR=true", in case you don't push to registry.
```

Comment:
```suggestion
Alternatively, you can trigger an on-demand default CR validation with `--validateCR=true`, in case you don't push to the registry.
```